.src/docker_build/main.py

```diff
@@ -8,6 +8,8 @@
 )
 from docker_build.validation import validate
 
+from settings import conf
```

Comment:
seems like `isort` hook is not working properly, cause `settings` should be located within the same block wit `docker_build`, and `typer` is a 3rd party. it's configurable via `setup.cfg`, but you could skip it for now, just FYI